docs/proposals/004-endpoint-picker-protocol/README.md

```diff
@@ -9,6 +9,15 @@ This doc defines the protocol between the EPP and the proxy (e.g, Envoy).
 The EPP MUST implement the Envoy
 [external processing service](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_proc/v3/ext_proc.proto) protocol.
 
+## Version History
+
+| Version  | Date       | Changes                                          |
+|----------|------------|--------------------------------------------------|
+| dev-main | 2025-07-29 | Added status metadata field for picked endpoints |
```

Comment:
@zhengkezhou1 this is a good start. Can you create a separate issue to add this table to https://gateway-api-inference-extension.sigs.k8s.io/guides/implementers/#callout-extension and tooling that automates the maintenance of the table? We may need to move the [EPP proto spec](https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/docs/proposals/004-endpoint-picker-protocol) outside of the `proposals` directory and treat it similarly to API types, e.g. something similar to `api-ref-docs` to maintain the spec.